assets/js/components/setup/search-console.js

```diff
@@ -134,6 +139,37 @@ class SearchConsole extends Component {
 		}
 	}
 
+	/**
+	 * Save Search Console property.
+	 * @param { string } siteURL
+	 */
+	async saveProperty( siteURL ) {
+		try {
+			const responseData = await data.set( 'modules', 'search-console', 'save-property', { siteURL } );
+
+			return new Promise( ( resolve ) => {
+				resolve( responseData );
+			} );
+		} catch ( err ) {
+			throw err;
+		}
```

Comment:
This is an unnecessary try/catch block since it does nothing else other than re-throwing the error. If there's nothing else to do in this case, it should be removed.